__tests__/buildTranslationFiles.spec.ts

```diff
@@ -278,11 +278,13 @@ describe.each(formats)('buildTranslationFiles in %s', (fileFormat) => {
 
       beforeEach(() => removeI18nFolder(type));
 
-      it('should work with service', () => {
+      it('should work with service (using constructor property and inject)', () => {
```

Comment:
Please sperate the inject into it's own case.

Reply:
I will.